reflection.md

```diff
@@ -2,10 +2,24 @@
 
 ## Which operators were easiest to implement and why?
 
+We thought that operators like ||, ~, and <*> were the easiest to implement because we just had to substitute in expressions from our RegularLanguage implementation where they were needed. All in all, it was a pretty straightforward task. We definitely had a lot more trouble with the operators that required a bit more thinking to get working!
+
 ## Which operators were most difficult to implement and why?
 
+The most difficult one was {n} becuase we encountered a weird error with our recursion. For some reason, Empty instead of epsilon as a base case cause runtime errors. Eventually we figured it out. Despite this, from the onset we had a pretty good idea of what we were supposed to do, and then we did it! We also had some trouble implementing the conversion from Strings to RegularLanguages. This is because we had to use some higher-order functions (map and reduce) together in a clever way. It took us some time, but understanding these higher-order functions well will help us in future assignments as well as our upcoming projects! 
```

Comment:
Same here, it was very difficult for me to implement this and I think the solution was unique so it was a useful learning experience for implementing DSLs with unique syntax.